vector/src/main/java/fr/gouv/tchap/activity/AccessibilityServiceDetectionActivity.java

```diff
@@ -0,0 +1,86 @@
+package fr.gouv.tchap.activity;
+
+
+import android.content.ContentResolver;
+import android.content.DialogInterface;
+import android.content.Intent;
+import android.os.Bundle;
+import android.provider.Settings;
+import android.support.annotation.Nullable;
+import android.support.v7.app.AlertDialog;
+import android.support.v7.app.AppCompatActivity;
+
+import org.matrix.androidsdk.util.Log;
+
+import im.vector.R;
+import im.vector.util.PreferencesManager;
+
+public class AccessibilityServiceDetectionActivity extends AppCompatActivity {
+
+    public static boolean isAccessibilityServiceActive(ContentResolver contentResolver) {
+        int accessibilityEnabled = 0;
+
+        try {
+            accessibilityEnabled = Settings.Secure.getInt(contentResolver, android.provider.Settings.Secure.ACCESSIBILITY_ENABLED);
+        } catch (android.provider.Settings.SettingNotFoundException e) {
+            Log.d("AccessibilityServiceDetectionActivity", "*** Security : accessibility settings not found ****");
+        }
+
+        return (accessibilityEnabled == 0) ? false : true;
+    }
+
+    @Override
+    protected void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        if (!PreferencesManager.detectAccessibilityService(this)) {
+            finish(true);
+            return;
+        }
+
+
+        if (!isAccessibilityServiceActive(getContentResolver())) {
+            finish(true);
+            return;
+        }
+
+        AlertDialog.Builder builder = new AlertDialog.Builder(this);
+        builder.setMessage(R.string.accessibility_security_dialog_message)
+                .setIcon(R.drawable.message_notification_transparent)
```

Comment:
maybe add `.setCancellable(false)` to force user to choose an option and not click on the area around the dialog to dismiss it (it's up to you to decide, that's just a suggestion)